fdbclient/SpecialKeySpace.actor.h

```diff
@@ -152,17 +151,17 @@ class SpecialKeySpace {
 private:
 	ACTOR static Future<Optional<Value>> getActor(SpecialKeySpace* sks, ReadYourWritesTransaction* ryw, KeyRef key);
 
-	ACTOR static Future<Standalone<RangeResultRef>> checkModuleFound(SpecialKeySpace* sks,
+	ACTOR static Future<Standalone<RangeResultRef>> checkRYWValid(SpecialKeySpace* sks,
```

Comment:
I changed the function name since it is now responsible for testing `ryw` liveness.